lib/api/core/plugins/pluginsManager.js

```diff
@@ -982,6 +995,7 @@ class PluginsManager {
 
       const
         plugin = {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Group all shorthand properties at either the beginning or end of this object declaration. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS3499)